DOCS/man/libmpv.rst

```diff
@@ -38,7 +38,7 @@ A C plugin must export the following function::
 
     int mpv_open_cplugin(mpv_handle *handle)
 
-The plugin function will be called on loading time. This function does not
+The plugin function will be called on loading time. This function must not
```

Comment:
This sounds weird, I think it was correct phrasing, that the function "does not" return.

Reply:
I thought it was weird to use a present tense statement to say the behavior that the user has to implement.

Reply:
After reading it again, this sentence doesn't make sense with both versions... How about we remove this sentence?

```
The plugin function will be called at load time. This function runs in a script-dedicated thread. The `handle` will be deallocated and the plugin unloaded as soon as the plugin function returns.
```

Reply:
Sure.